### DISCUSSION TITLE
Disable linking to rdrr.io?

### DISCUSSION THREAD

**Discussion by adamancer** [carpentries/workbench#14] [Q&A] [answered]
I'm trying to use the workbench to develop a python lesson and ran into an issue with automatic linking. Certain backticked functions with common names (like min() or max()) automatically get linked to rdrr.io when a site is generated. This is obviously confusing for a python lesson--is there any way to disable the links? I haven't been able to figure out where they're coming from.

**Comment by zkamvar** [comment, marked as answer]
Hi @adamancer, thank you for the feedback! 

I agree that these links make no sense in this context. I've opened https://github.com/carpentries/sandpaper/issues/298 and I will try to get to it as soon as I can :)

**Comment by zkamvar** [comment]
Hi @adamancer, I'm working on this now, but I can not seem to reproduce the issue locally or on a test website: https://zkamvar.github.io/potential-porpoise/01-introduction.html#introduction

In this example, I [added a single line with known functions names from R](https://github.com/zkamvar/potential-porpoise/commit/4d024e547636feb219653031225b7537a6ff6c7b), but [the output is not linked](https://github.com/zkamvar/potential-porpoise/blob/219182f4267247bc8c42b4aed0d8867d9d7c50c4/01-introduction.html#L239).

Can you show me your site where you are seeing this?

**Reply by adamancer** [reply]
Hi @zkamvar,

Thanks for taking a look at this. [This is the lesson](https://adamancer.github.io/python-ecology-lesson-alt/index.html), but you're right, the links no longer appear. They did show up when I built the site locally, but it looks like that was because my copy of sandpaper was out of date. Sorry about that! You can close the ticket.

**Reply by zkamvar** [reply]
Thank you for the followup and no worries about the false alarm, Given the rapid nature of development on The Workbench, it's completely understandable that your local version would be out of date 😄
